lib/mp-readline/readline.c

```diff
@@ -214,10 +247,40 @@ int readline_process_char(int c) {
             redraw_step_forward = 1;
         }
     } else if (rl.escape_seq == ESEQ_ESC) {
+        size_t num_chars;
```

Comment:
This should be a local variable, local in scope to the case's that use it.

Reply:
I had initially tried to do this but the compiler was complaining at me, but have since figured out that I need to wrap the `case` body in `{ ... }`.